spec/rails_helper.rb

```diff
@@ -38,6 +56,7 @@ def login_user(user)
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.include(OmniauthMacros)
```

Comment:
Maybe It makes sense to only include the module in tests where it's actually used? Instead of all the specs?